concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/contractexample/cis2nft/Cis2NftParameters.java

```diff
@@ -0,0 +1,140 @@
+package com.concordium.sdk.examples.contractexample.cis2nft;
+
+import com.concordium.sdk.examples.contractexample.parameters.*;
+import com.concordium.sdk.transactions.ReceiveName;
+import com.concordium.sdk.transactions.smartcontracts.Schema;
+import com.concordium.sdk.transactions.smartcontracts.SchemaParameter;
+import com.concordium.sdk.transactions.smartcontracts.SchemaVersion;
+import com.concordium.sdk.types.*;
+import lombok.SneakyThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Helper class for creating and initializing parameters being used in {@link Cis2Nft}.
+ */
+public class Cis2NftParameters {
+
+    private static final AccountAddress ACCOUNT_ADDRESS = AccountAddress.from("3XSLuJcXg6xEua6iBPnWacc3iWh93yEDMCqX8FbE3RDSbEnT9P");
+    private static  final ContractAddress CONTRACT_ADDRESS_1 = ContractAddress.from(1, 0);
+    private static final ContractAddress CONTRACT_ADDRESS_2 = ContractAddress.from(2, 0);
+
+    private static final String CIS_2_NFT_CONTRACT_NAME = "cis2_nft";
+
+    @SneakyThrows
+    public static SchemaParameter generateMintParams() {
+        Schema schema = Schema.from(Files.readAllBytes(Paths.get("./src/main/java/com/concordium/sdk/examples/contractexample/cis2nft/cis2-nft.schema.bin")), SchemaVersion.V3);
+        ReceiveName mintParamsReceiveName = ReceiveName.from(CIS_2_NFT_CONTRACT_NAME, "mint");
+        List<TokenIdU32> tokens = new ArrayList<>();
+        tokens.add(TokenIdU32.from(2));
+        tokens.add(TokenIdU32.from(22));
+        tokens.add(TokenIdU32.from(2132));
+        SchemaParameter mintParameter = new MintParams(schema, mintParamsReceiveName, ACCOUNT_ADDRESS, tokens);
+        mintParameter.initialize(true);
+        return mintParameter;
+    }
+
+    @SneakyThrows
+    public static SchemaParameter generateTransferParams() {
```

Comment:
Please add a bit context to each of these functions.

Reply:
<img width="772" alt="image" src="https://github.com/Concordium/concordium-java-sdk/assets/93247743/7c11e651-5671-43ca-8acb-35faa2a7f917">

I have done something like this for every method. Does it need to be more detailed?